.github/workflows/ci.yaml

```
@@ -32,11 +32,15 @@ jobs:
         flake8
     - name: Test with pytest
       run: |
-        PYTHONPATH=./src:$PYTHONPATH coverage run -m pytest -r sx
-    - name: Report coverage with Codecov
-      if: github.event_name == 'push'
-      run: |
-        codecov --token=${{ secrets.CODECOV_TOKEN }}
+        pip install pytest-cov
+        export PYTHONPATH=./src:$PYTHONPATH
+        pytest --cov=./tests --cov-report=xml
```

I believe that the `--cov` argument should take the path to the source code, not the tests:
```suggestion
        pytest --cov=./src --cov-report=xml
```